Enterprise docker getting started fixes (#12346)
gravitational/teleport: docs/pages/enterprise/getting-started.mdx

### Systemd Unit File

> Next, download the systemd service unit file from [examples directory](https://github.com/gravitational/teleport/tree/(=teleport.version=)/examples/systemd)

Reviewer: ```suggestion
Download the systemd unit file for Teleport from the [examples directory](https://github.com/gravitational/teleport/tree/v(=teleport.version=)/examples/systemd)
```

### Systemd Unit File

Download the systemd unit file for Teleport from the [examples directory](https://github.com/gravitational/teleport/tree/v(=teleport.version=)/examples/systemd)

> on Github and save it as `/etc/systemd/system/teleport.service` on both servers.

Reviewer: Not sure why this change is being made

Author: fixed the camelcase of GitHub @ptgott

  You will need a recent version of [`docker-compose`](https://docs.docker.com/compose/install/) installed to follow this section of the quick start guide.
</Admonition>

> The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:

Reviewer: ```suggestion
The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/v(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:
```

```code

> # download your license file from the Gravitational dashboard and put it in the correct directory

Reviewer: ```suggestion
# Download your license file from the Gravitational dashboard and put it in the correct directory.
```

> # the file needs to be named license.pem

Reviewer: ```suggestion
# The file needs to be named "license.pem".
```

$ cp ~/downloads/license.pem .

# download the quickstart file from our Github repo

> $ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/(=teleport.version=)/docker/teleport-ent-quickstart.yml

Reviewer: ```suggestion
$ curl -Lso teleport-ent-quickstart.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-ent-quickstart.yml
```

  You will need a recent version of [`docker-compose`](https://docs.docker.com/compose/install/) installed to follow this section of the quick start guide.
</Admonition>

The easiest way to start Teleport Enterprise quickly is to use `docker-compose` with our [`teleport-ent-quickstart.yml`](https://github.com/gravitational/teleport/blob/v(=teleport.version=)/docker/teleport-ent-quickstart.yml) file:

```code
# Download your license file from the Gravitational dashboard and put it in the correct directory.
# The file needs to be named "license.pem".
$ cp ~/downloads/license.pem .

> # Download the quickstart file from our Github repo

Reviewer: ```suggestion
# Download the quickstart file from our GitHub repo
```
